keys/constants.go

```diff
@@ -177,7 +173,9 @@ var (
 	StatusStorePrefix = MakeKey(StatusPrefix, proto.Key("store-"))
 	// StatusNodePrefix stores all status info for nodes.
 	StatusNodePrefix = MakeKey(StatusPrefix, proto.Key("node-"))
-	// TableDataPrefix prefixes all Table data to aid in transitioning
-	// key:value data to Table data, and for ease of debugging.
-	TableDataPrefix = proto.Key("table-")
+
+	// TableDataPrefix prefixes all table data. It is specifically chosen to
+	// occur after the range of common user data prefixes so that tests which use
+	// those prefixes will not see table data.
+	TableDataPrefix = proto.Key("\xff")
```

Comment:
I thought this had a bit of symmetry with the `SystemPrefix` of `\x00`. The table keys aren't meant for human consumption and they will be the majority of data in the system, so it seems worthwhile to keep this prefix at a single byte. Another option would be to use the prefix `\x01`. I don't see an obvious choice between the two.